docs/source/examples/checks/methodology/single_feature_contribution_train_test.ipynb

```diff
@@ -5,123 +5,83 @@
    "id": "21c5d5bc",
```

Comment:
<p>We can define on our check a condition that will validate that our pps scores aren't too high. The check has 2 possible built-in conditions:</p><p> </p><p>--</p><p>Let's add the conditions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>